### PR TITLE
Use `<TextInput>` component

### PR DESCRIPTION
**Pull request by vadimdemedes**
Simpler code, happier developers.

### CODE REVIEW

**Comment by sindresorhus** [comment]
The circular arrow selection no longer works.

File: ui.js [hunk 1]
```diff
@@ -191,9 +199,7 @@ class Emoj extends Component {
 			return;
 		}
 
-		if (key.name === 'backspace') {
-			query.pop();
-		} else if (key.name === 'return' || (key.ctrl && key.name === 'u')) {
+		if (key.name === 'return' || (key.ctrl && key.name === 'u')) {
```
Comment (sindresorhus) [inline ui.js]: You can remove the `|| (key.ctrl && key.name === 'u')` part now.

**Comment by vadimdemedes** [comment]
Fixed all.

**Comment by sindresorhus** [comment]
Purrfect

**Pull request merged**